lib/plugins/aws/package/compile/events/cloudFront/index.test.js

```diff
@@ -1769,3 +1748,196 @@ describe('#AwsCompileCloudFrontEvents() package', () => {
     });
   });
 });
+
+describe('#AwsCompileCloudFrontCachePolicies() package', () => {
+  const fixtureName = 'functionCloudFront';
+  const edgeFunctionName = 'foo';
+  const cachePolicyName = 'allInCache';
+  const cachePolicyConfig = {
+    DefaultTTL: 60,
+    MinTTL: 30,
+    MaxTTL: 3600,
+    ParametersInCacheKeyAndForwardedToOrigin: {
+      CookiesConfig: {
+        CookieBehavior: 'whitelist',
+        Cookies: ['my-public-cookie'],
+      },
+      EnableAcceptEncodingBrotli: true,
+      EnableAcceptEncodingGzip: true,
+      HeadersConfig: {
+        HeaderBehavior: 'whitelist',
+        Headers: ['authorization', 'content-type'],
+      },
+      QueryStringsConfig: {
+        QueryStringBehavior: 'allExcept',
+        QueryStrings: ['not-cached-query-string'],
+      },
+    },
+  };
+  function extendAndPackage(extension) {
+    return runServerless({ fixture: fixtureName, configExt: extension, cliArgs: ['package'] });
```

Comment:
To optimize tests performance, it'll be nice to minimize count of `runServerless` runs (so we tests as much as we could with single `runServeless` run). 

Do you think we can refactor those tests into one `runServerless` run (or simply a minimal required count) ?

e.g. ALB health check tests, provide a good example: https://github.com/serverless/serverless/blob/80e70e7affd54418361c4d54bdef1561af6b8826/lib/plugins/aws/package/compile/events/alb/lib/healthCheck.test.js